android/src/ti/map/TiUIMapView.java

```diff
@@ -865,6 +865,10 @@ public void onMapClick(LatLng point) {
 		}
 		clickablePolylines.clear();
 
+		KrollDict d = new KrollDict();
+		d.put(TiC.PROPERTY_LATITUDE, point.latitude);
+		d.put(TiC.PROPERTY_LONGITUDE, point.longitude);
+		proxy.fireEvent("mapclick", d);
```

Comment:
Ok, so `source` already includes it. I think other iOS events have the `map` one as well, wondering if Android has it there.